include/pisa/bit_vector.hpp

```diff
@@ -19,7 +19,7 @@ class bit_vector_builder {
 
     bit_vector_builder(uint64_t size = 0, bool init = false) : m_size(size)
     {
-        m_bits.resize(detail::words_for(size), uint64_t(-static_cast<int>(init)));
+        m_bits.resize(detail::words_for(size), init ? std::numeric_limits<std::uint64_t>::max() : 0U);
```

Comment:
why? I believe this is a different behaviour

Reply:
Looks correct to me: https://godbolt.org/z/GPEg2g what behavior are you expecting?

Reply:
I didn’t realize it was a bool. I thought it was an unsigned int